tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js

```diff
@@ -0,0 +1,106 @@
+import chai from 'chai'
+import {openAndStartCensusQuestionnaire} from '../../../helpers'
+
+import PermanentOrFamilyHome from '../../../pages/surveys/census/household/permanent-or-family-home.page.js'
```

Comment:
Yeh i guess so, they are auto generated with the page models. Have we excluded tests from any code checking? as those things normally go nuts over this.